docs/proposals/caching.adoc

```diff
@@ -0,0 +1,77 @@
+= Proposal: Caching
+
+== Purpose
+
+This proposal aims to support caching of dependencies
+(https://github.com/opendevstack/ods-pipeline/issues/147) in order to speed up build times. 
+
+== Background
+
+Currently there is a single PVC per project which is shared by all build pipelines. 
+As a consequence only a single build at a time can run for a project.
```

Comment:
Technically this isn't actually true due to #394, but with #407 and #160 solved, it will be "one build at a time for a repository".

Reply:
I will add this clarification to the proposal so we don't loose it when resolving the conversation.